tag/nitag.yml

```
@@ -1663,6 +1663,7 @@ definitions:
         path: foo
         type: DOUBLE
         keywords: [fooKeyword, barKeyword]
+        last_updated: 1696579890
```

Shouldn't this be camelCase and ISO-formatted?

```suggestion
        lastUpdated: '2023-10-06T11:45:08Z'
```